Gemfile

```diff
@@ -45,15 +45,14 @@ gem 'ffi-icu'
 gem 'unicode'
 gem 'rack-openid'
 gem 'ruby-openid'
-gem 'secure_headers'
 gem 'minitest-rails'
 gem 'rake-progressbar'
 gem 'rails_autolink'
 gem 'flag_shih_tzu'
 gem 'wicked_pdf'
 gem "font-awesome-rails"
 gem 'friendly_id', '~> 5.1.0'
-gem 'auto_html'
+gem 'auto_html', '~>1.6.4'
```

Comment:
:+1: The current code breaks with version 2 of that gem, so it's a good idea to constrain the version in the Gemfile.